From ed557af024914cf1359647ad36669d9679426198 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 5 Jan 2023 17:27:44 +0100
Subject: [PATCH 396/529] ARM: dts: stm32: default disable RNG on STM32MP15x
 boards

Default disables the rng1 node as this device is secured by default
for STM32MP15x boards, therefore managed by OP-TEE.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ia13659f3af1a93cf4add89d5f131d5858166fe47
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283443
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157a-ed1.dts  | 4 ----
 arch/arm/boot/dts/stm32mp157c-ed1.dts  | 4 ----
 arch/arm/boot/dts/stm32mp157d-ed1.dts  | 4 ----
 arch/arm/boot/dts/stm32mp157f-ed1.dts  | 4 ----
 arch/arm/boot/dts/stm32mp15xx-dkx.dtsi | 4 ----
 5 files changed, 20 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157a-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ed1.dts
@@ -351,10 +351,6 @@
 	vdd_3v3_usbfs-supply = <&vdd_usb>;
 };
 
-&rng1 {
-	status = "okay";
-};
-
 &rtc {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157c-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ed1.dts
@@ -363,10 +363,6 @@
 	vdd_3v3_usbfs-supply = <&vdd_usb>;
 };
 
-&rng1 {
-	status = "okay";
-};
-
 &rtc {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157d-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ed1.dts
@@ -359,10 +359,6 @@
 	vdd_3v3_usbfs-supply = <&vdd_usb>;
 };
 
-&rng1 {
-	status = "okay";
-};
-
 &rtc {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157f-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157f-ed1.dts
@@ -363,10 +363,6 @@
 	vdd_3v3_usbfs-supply = <&vdd_usb>;
 };
 
-&rng1 {
-	status = "okay";
-};
-
 &rtc {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
+++ b/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
@@ -518,10 +518,6 @@
 	vdd_3v3_usbfs-supply = <&vdd_usb>;
 };
 
-&rng1 {
-	status = "okay";
-};
-
 &rtc {
 	status = "okay";
 };
